javascript/javascript1/week3/homework.js

```diff
@@ -0,0 +1,199 @@
+// Item array removal
+
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+];
+
+const nameToRemove = "Ahmad";
+const index = names.indexOf(nameToRemove);
+names.splice(index, 1);
+console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+// When will we be there??
+
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+};
+
+function calcTime(travelInformation) {
+
+    const time = travelInformation.destinationDistance / travelInformation.speed;
+    var decimalTime = parseFloat(time);
+    decimalTime = decimalTime * 60 * 60;
+    var hours = Math.floor((decimalTime / (60 * 60)));
+    decimalTime = decimalTime - (hours * 60 * 60);
+    var minutes = Math.floor((decimalTime / 60));
+    return hours + " hours and " + minutes + " minutes";
+
+}
+const travelTime = calcTime(travelInformation);
+console.log(travelTime); // 8 hours and 38 minutes
+
+// Series duration of my life
+
+const seriesDurations = [
+    {
+        title: "Game of thrones",
+        days: 3,
+        hours: 1,
+        minutes: 0,
+    },
+    {
+        title: "One Punch Man",
+        days: 0,
+        hours: 9,
+        minutes: 36,
+    },
+    {
+        title: "One Piece",
+        days: 16,
+        hours: 17,
+        minutes: 14,
+    },
+];
+
+const date1 = new Date(1996, 1, 13);
+const date2 = new Date(date1.getFullYear() + 80, 1, 13);
+const seconds = Math.floor((date2 - (date1)) / 1000);
```

Comment:
There are some extra brackets here.
```suggestion
const seconds = Math.floor((date2 - date1) / 1000);
```